src/drive/mobile/modules/settings/components/SelectMediaBuckets.jsx

```diff
@@ -0,0 +1,108 @@
+/* global cozy */
+import React from 'react'
+import { connect } from 'react-redux'
+import Main from 'drive/web/modules/layout/Main'
+import ScrollToTop from 'drive/web/modules/navigation/ScrollToTop'
+import styles from '../styles'
+import Toggle from 'cozy-ui/react/Toggle'
+import { getMediaBuckets, addMediaBucket, delMediaBucket } from '../duck'
+import { getMediaBuckets as getAllMediaBuckets } from 'drive/mobile/lib/media'
+
+const { BarCenter } = cozy.bar
+
+class SelectMediaBuckets extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      mediaBuckets: []
```

Comment:
since we use add() and delete(), we should declare our initial state as a Set()

Reply:
Fixed